packages/components/Spinner/ios/MSFSpinnerViewManager.m

```diff
@@ -0,0 +1,42 @@
+#import <React/RCTComponent.h>
+#import <React/RCTViewManager.h>
+
+@import FluentUI;
+
+@interface MSFActivityIndicatorView(RCTComponent)
+
+@end
+
+@implementation RCTConvert (MSFActivityIndicatorView)
+
+RCT_ENUM_CONVERTER(MSFActivityIndicatorViewSize, (@{
+	@"xSmall": @(MSFActivityIndicatorViewSizeXSmall),
+	@"small": @(MSFActivityIndicatorViewSizeSmall),
+	@"medium": @(MSFActivityIndicatorViewSizeMedium),
+	@"large": @(MSFActivityIndicatorViewSizeLarge),
+	@"xLarge": @(MSFActivityIndicatorViewSizeXLarge),
+
+}), MSFActivityIndicatorViewSizeSmall, integerValue);
+
+@end
+
+@interface RCT_EXTERN_MODULE(MSFSpinnerViewManager, RCTViewManager)
+
+RCT_EXPORT_VIEW_PROPERTY(hidesWhenStopped, BOOL);
+
```

Comment:
Nit- Can probably delete the extraneous line in between the hidesWhenStopped export and the color expo